pyproject.toml

```diff
@@ -24,6 +24,8 @@ addopts = "-ra -v"
 
 [tool.mypy]
 
+python_version = "3.7"
```

Comment:
I'm nitpicking, but can you please move this down below? For example, something like this to follow the mypy config docs:

```
... snip ...
# Import Discovery
ignore_missing_imports = false

# Platform configuration
python_version = "3.7"

# Disallow dynamic typing
disallow_any_unimported = true
disallow_any_expr = false
... snip ...
```